boards/arm/nucleo_wb55rg/nucleo_wb55rg.dts

```diff
@@ -22,6 +22,20 @@
 		zephyr,code-partition = &slot0_partition;
 	};
 
+	power-states {
+		idle: idle {
+			compatible = "zephyr,power-state";
+			power-state-name = "suspend-to-idle";
+			min-residency-us = <1000>;
+		};
+
+		standby: standby {
```

Comment:
This doesn't seem entirely consistent with https://github.com/zephyrproject-rtos/zephyr/issues/31162#issuecomment-756818940, though it might be a better mapping.  It could help, especially at this early stage, if these had a comment specifying which STM32 modes they mapped to.

Reply:
Right, we'll use STM32 dedicated names.